internal/logic/logic_test.go

```diff
@@ -120,3 +131,456 @@ func Test_htmlPdfServiceLogic_Ping(t *testing.T) {
 		})
 	}
 }
+
+//func Test_Upload(t *testing.T) {
```

Comment:
Why are the test cases for this commented?

Reply:
done